pkg/rpc/server/library/server.go

```diff
@@ -39,7 +39,7 @@ func (s *Server) Detect(_ context.Context, req *proto.LibDetectRequest) (res *pr
 		return nil, err
 	}
 
-	s.vulnClient.FillInfo(vulns, false)
+	s.vulnClient.FillInfo(vulns, "", false)
```

Comment:
TODO: fill ReportType

Reply:
Yeah, where can we get this from?

Reply:
Also shouldn't a test be failing right now if this information is missing?

Reply:
fixed https://github.com/aquasecurity/trivy/pull/433/commits/fc2eed5def774ca6d0a6e3daa59f865d2cfa83c3